download.html

```diff
@@ -11,11 +11,25 @@
 form label {
 	display: block;
 	padding: .2em;
+	padding-left: 1.9em;
+	page-break-inside: avoid;
+	break-inside: avoid;
+	-webkit-column-break-inside: avoid;
 }
 
-form input {
-	margin-right: .7em;
-}
+	form label > *:nth-child(2) {
+		margin-right: .3em;
+	}
+
+	form label a.owner {
+		margin-left: 0;
+		hyphens: none;
+	}
+
+	form label input {
+		margin-right: .7em;
+		margin-left: -1.7em;
+	}
```

Comment:
The  CSS you added appears indented further than the code around it.

Reply:
Well, because it is about elements inside `form label`, so it gets indented.
This practice can also be seen further down below.

Reply:
Ah ok, that works.